packages/ui/components/Form/Inputs/TextInput.tsx

```diff
@@ -0,0 +1,22 @@
+
+export type TextInputProps = {
+  placeholder: string
+  type?: 'text' | 'number' | 'email'
+  value?: string
+  onChange?: () => void
+  isDisabled?: boolean
+  className?: string
+}
+
+export const TextInput = (props: TextInputProps) => {
+  return (
+    <input
+      {...props}
+      disabled={props.isDisabled}
```

Comment:
```suggestion
      disabled={props.isDisabled}
      aria-label={props.placeholder}
```

Reply:
I don't think an aria-label is needed here but I may be wrong. In any case, if we choose to have an aria-label here, we should have an option to override it:

```suggestion
      disabled={props.isDisabled}
      aria-label={props.ariaLabel || props.placeholder}
```

Reply:
I agree with @fmvilas. no aria-label is needed here. since each form elements has a label attached to it with `htmlFor` attribute.